wpilibc/src/main/native/include/frc/controller/ProfiledPIDController.h

```diff
@@ -292,9 +293,34 @@ class ProfiledPIDController
   }
 
   /**
-   * Reset the previous error, the integral term, and disable the controller.
+   * Reset the previous error and the integral term.
+   *
+   * @param measurement The current measured State of the system.
+   */
+  void Reset(const State measurement) {
```

Comment:
Should be `const State& measurement`. There is no need to pass by value here since we are copying the object anyway in Line 302.